client/js/components/user/CustomerSettings/CustomerSettingsContact.vue

```diff
@@ -0,0 +1,241 @@
+<template>
+  <div class="layout u-pl">
+    <dp-editable-list
+      ref="editableList"
+      :entries="contacts"
+      :translation-keys="translationKeys"
+      @reset="resetForm"
+      @saveEntry="id => dpValidateAction('contactData', () => createOrUpdateContact(id), false)">
+      <template v-slot:list="contact">
+        <p
+          class="weight--bold u-mt"
+          v-text="contact.attributes.title" />
+        <span
+          class="block"
+          v-text="contact.attributes.phoneNumber" />
+        <span
+          class="block"
+          v-text="contact.attributes.eMailAddress" />
+        <span
+          class="block"
+          v-html="contact.attributes.text" />
```

Comment:
Why do You use a `span.block` and not a `p`? P would be much better for semantic HTML
(here and above)

Reply:
Because I wanted to move the edit and delete icons next to the last entry and `float: left` did strange things. I agree that semantic wise it's not good. Maybe @elmasdemos has an idea?